### PR TITLE
Integrate gutenberg-mobile release v1.112.0

### PR DESCRIPTION
**Pull request by derekblank**
## Description
This PR incorporates the 1.112.0 release of gutenberg-mobile.
For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/6597

Release Submission Checklist

- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is not assigned to a milestone.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by fluiddot — COMMENTED** [review]
Following the Release Submission Checklist in the PR:
> I have considered if this change warrants user-facing release notes and have added them to RELEASE-NOTES.txt if necessary.

It would be interesting to check if this release introduced user-facing changes (e.g. _Fix crash occurring on large post on Android_) and if so, reflect it.

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

**NOTE:** Before merging this PR, we need to update the Gutenberg Mobile reference to point to the release tag. Besides, we could review the release notes as mentioned in https://github.com/wordpress-mobile/WordPress-iOS/pull/22513#pullrequestreview-1856374085.

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/wordpress-mobile/WordPress-iOS/milestone/269">24.2</a>. This milestone is due in less than 4 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 24.2
 -----
+* [**] Prevent images from temporarily disappearing when uploading media [https://github.com/WordPress/gutenberg/pull/57869]
+* [**] Fix crash occurring on large post on Android [https://github.com/WordPress/gutenberg/pull/58266]
```
Comment (fluiddot) [inline RELEASE-NOTES.txt:4]: @derekblank Similar to https://github.com/wordpress-mobile/WordPress-Android/pull/20093#pullrequestreview-1858712761, we could add more detail to the release notes by adding the prefix `Block Editor: ` for entries related to the editor ([example](https://github.com/wordpress-mobile/WordPress-iOS/blob/8e8c5b41a7ae9d2c6a20d4bcf411506243554ee2/RELEASE-NOTES.txt#L13)).

Comment (fluiddot) [reply RELEASE-NOTES.txt:4]: I'm afraid I completely overlooked that this entry shouldn't be included as it's an Android-only one. Fortunately, it was fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/22562. Thanks 🙇 !

**Pull request merged**